java/README.md

```diff
@@ -130,6 +130,22 @@ Maven:
 </dependency>
 ```
 
+SBT:
+- **IMPORTANT** must include a `classifier`. Please use this dependency block and add it to the build.sbt file.
```

Comment:
Why is the classifier is a must? shouldn't SBT decide based on the running platform?

Reply:
It's the same as for Maven and Gradle, which both also have this note.  There are plugins that can do this for each of the 3 build tools, but on its own, SBT cannot do this.